frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -684,6 +689,20 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                       }
                     : undefined
 
+                const includeRevenue = !!featureFlags[FEATURE_FLAGS.WEB_REVENUE_TRACKING]
+
+                const revenueEventsSeries: EventsNode[] | undefined =
+                    includeRevenue && currentTeam?.revenue_tracking_config
+                        ? currentTeam.revenue_tracking_config.events.map((e) => ({
+                              math: PropertyMathType.Sum,
+                              name: e.eventName,
+                              event: e.eventName,
+                              kind: NodeKind.EventsNode,
+                              custom_name: e.eventName,
+                              math_property: e.revenueProperty,
+                          }))
+                        : undefined
```

Comment:
To simplify types, could just make this an empty array rather than undefined, the rest of the code will work just fine because you're checking the length

Reply:
That makes sense for now - I think we'll probably change this around later as if they don't have any revenue events, we'll probably want to replace the graph with a link to settings